src/nodes.coffee

```diff
@@ -2911,6 +2928,20 @@ exports.ClassProperty = class ClassProperty extends Base
       operator: @operatorToken?.value ? '='
       staticClassName: @staticClassName?.ast(o) ? null
 
+exports.ClassPrototypeProperty = class ClassPrototypeProperty extends Base
+  constructor: ({@name, @value}) ->
+    super()
+
+  children: ['name', 'value']
+
+  isStatement: YES
+
+  astProperties: (o) ->
```

Comment:
This mimics the properties of Babel AST types like `ClassProperty`/`ObjectProperty`